frontend/src/employee-frontend/components/common/Filters.tsx

```diff
@@ -112,7 +112,7 @@ export function Filters({
   const { i18n } = useTranslation()
   return (
     <FiltersContainer>
-      {setFreeText && searchPlaceholder && (
+      {setFreeText && !!searchPlaceholder && (
```

Comment:
Not related to your changes, but this condition just seems wrong. Hiding the whole input if a placeholder is missing or empty 😄 

Could you fix this as well?

Reply:
Sure! 😄